Podfile

```diff
@@ -95,14 +95,6 @@ target 'AzureStorageBlob' do
   end
 end
 
-target 'AzureTemplate' do
-  project 'sdk/template/AzureTemplate/AzureTemplate'
-
-  target 'AzureTemplateTests' do
-    inherit! :search_paths
-  end
-end
-
```

Comment:
Why is this being removed?

Reply:
I don't know why but the changes to the Podfile that we made a while back when we had our merge conflict keep coming back. So I have to keep on removing them. This is like the third time. If you look up in the file it was defined twice, and this one is the one with the error because `AzureTemplateTests` does not exist on our branch of AzureTest.